slices/libasan6.yaml

```
@@ -0,0 +1,9 @@
+package: libasan6
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libstdc++6_libs
+    contents:
+      /usr/lib/x86_64-linux-gnu/libasan.so*:
```

```suggestion
      /usr/lib/*-linux-*/libasan.so.6*:
```

You need the first two `*` to be able to cope with other architectures.